docs/versions/v26.4.5/guide/build/runtime-layouts/function-providers.md

```
@@ -0,0 +1,206 @@
+| Provider | Test Class | Requirements |
+|----------|-----------|--------------|
+| AWS Lambda | `*-EndToEndIT` | AWS credentials, SAM/CloudFormation |
+| Azure Functions | `AzureFunctionsEndToEndIT` | Azure subscription, Terraform |
+| GCP Cloud Functions | `GcpFunctionsBootstrapSmokeTest` | GCP project, Functions Framework |
+
+## Troubleshooting
+
+### Common Issues
+
+**Multiple providers detected:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)